AUTHORS

```diff
@@ -1,157 +1,338 @@
+  igor.ivanov.va@gmail.com
+  igor.ivanov@itseez.com
+Igor Usarov, Mellanox
+  igoru@mellanox.com
+Jeff Squyres, University of Indiana, Cisco
+  jeff@squyres.com
+  jsquyres@cisco.com
+  jsquyres@users.noreply.github.com
+Jelena Pjesivac-Grbovic, University of Tennessee-Knoxville
+  pjesa@icl.iu.edu
+Jithin Jose, Intel
+  jithin.jose@intel.com
+John Westlund, Intel
+  john.a.westlund@intel.com
+Jon Mason, OpenGrid Computing
+  jdmason@opengridcomputing.com
+Jose Roman, Universitat Politecnica de Valencia
+  jroman@dsic.upv.es
+Josh Hursey, Indiana University, Oak Ridge National Laboratory, Los Alamos National Laboratory, Lawrence Berkeley National Laboratory, University of Wisconsin-La Crosse, IBM
+  jhursey@us.ibm.com
+  jjhursey@open-mpi.org
+Joshua Ladd, Mellanox
+  jladd.mlnx@gmail.com
+  joshual@mellanox.com
+KAWASHIMA Takahiro, Fujistu
+  t-kawashima@jp.fujitsu.com
+Karen Norteman, Sun
+  karen.norteman@sun.com
+Karol Mroz, University of British Columbia
+  mroz.karol@gmail.com
+Kenneth Matney, Oak Ridge National Laboratory
+  matneykdsr@ornl.gov
+L. R. Rajeshnarayanan, Intel
+  l.r.rajeshnarayanan@intel.com
+Laura Casswell, Los Alamos National Laboratory
+  lcasswell@lanl.gov
+Lenny Verkhovsky, Volataine
+  lennyb@voltaire.com
+Leobardo Ruiz Rountree, Individual
+  lruizrountree@gmail.com
+Li-Ta Lo, Los Alamos National Laboratory
+  ollie@lanl.gov
+MPI Team (bot), self
+  mpiteam@open-mpi.org
+Mangala Jyothi Bhaskar, University of Houston
+  mjbhaskar@crill.cs.uh.edu
+  mjbhaskar@salmon.cs.uh.edu
+  mjbhaskar@uh.edu
+Manjunath Gorentla Venkata, Oak Ridge National Laboratory
+  manjugv@ornl.gov
+Mark Santcroos, Rutgers University
+  m.a.santcroos@amc.uva.nl
+  mas781@scarletmail.rutgers.edu
+Mark Taylor, Los Alamos National Laboratory
+  mt@lanl.gov
+Matias A Cabral, Intel
+  matias.a.cabral@intel.com
+  matcabral@users.noreply.github.com
+Matthias Jurenz, Technische Universitaet Dresden
+  matthias.jurenz@tu-dresden.de
+Maximilien Levesque, Individual
+  maximilien.levesque@gmail.com
+Mike Dubman, Mellanox
+  miked@mellanox.com
+Mitch Sukalski, Sandia National Laboratory
+  mwsukal@ca.sandia.gov
+Mohamad Chaarawi, University of Houston
+  mschaara@cs.uh.edu
+Nadezhda Kogteva, Mellanox
+  nadezhda@mngx-orion-01.dmz.e2e.mlnx
+  nadezhda.kogteva@itseez.com
+Nadia Derbey, Bull
+  Nadia.Derbey@bull.net
+Nathan Hjelm, Los Alamos National Laboratory
+  hjelmn@cs.unm.edu
+  hjelmn@lanl.gov
+  hjelmn@me.com
+Nathaniel Graham, Los Alamos National Laboratory
+  ngraham@lanl.gov
+  nrgraham23@gmail.com
+Nick Papior Andersen, Individual
+  nickpapior@gmail.com
+Nysal Jan K A, IBM
+  jnysal@gmail.com
+  jnysal@in.ibm.com
+Orion Poplawski, Individual
+  orion@cora.nwra.com
+Oscar Vega-Gisbert, Universitat Politecnica de Valencia
+  ovega@dsic.upv.es
+Pak Lui, Sun
+  pak.lui@sun.com
+Patrick Geoffray, Myricom
+  patrick@myri.com
+Pavel Shamis, Mellanox, Oak Ridge National Laboratory
+  shamisp@ornl.gov
+  shamisp@users.noreply.github.com
```

Comment:
@shamisp We're only going by the email addresses that you have used in existing commits.  You haven't used your ARM address yet (i.e., it doesn't make sense to list ARM for you yet, because there's no commits from you while at ARM yet), nor the pasharesearch address.

To be clear: we can't change the email address that you used in past commits.